01week/rockPaperScissors.js

```diff
@@ -9,9 +9,22 @@ const rl = readline.createInterface({
 
 
 function rockPaperScissors(hand1, hand2) {
-
-  // Write code here
-
+    hand1 = hand1.toLowerCase(); //changes all letters to lower case
+    hand2 = hand2.toLowerCase();
+    hand1 = hand1.trim(); //trims whitespace from beginning and end of string
+    hand2 = hand2.trim();
```

Comment:
To refactor this code and make it 2 line instead of 4 you can chain the methods. For example : hand1.toLowerCase().trim(); 
all in one!